OrbitGl/App.h

```diff
@@ -281,7 +278,10 @@ class OrbitApp final : public CoreApp, public DataViewFactory {
   std::vector<std::string> m_SymbolDirectories;
   std::function<void(const std::string&)> m_UiCallback;
 
-  std::vector<std::shared_ptr<struct Module>> m_ModulesToLoad;
+  absl::flat_hash_set<std::shared_ptr<struct Module>> modules_to_load_;
```

Comment:
Is `struct` necessary?

Reply:
nope, forget to delete it. Don't know why it was there in the first place